packages/tests/tests/store/cursor.node.spec.ts

```diff
@@ -170,7 +173,7 @@ describe("Waku Store, cursor", function () {
       if (
         !(err instanceof Error) ||
         !err.message.includes(
-          "Store query failed with status code: 300, description: BAD_RESPONSE: archive error: DRIVER_ERROR: cursor not found"
+          "Store query failed with status code: 300, description: BAD_RESPONSE: archive error: DIRVER_ERROR: cursor not found"
```

Comment:
This is strange, I assumed I corrected the typo locally and hence why I would see the error because what matter is what nwaku node returns. I will see if I can change the test to be passing no matter if typo is present.

Reply:
This was fixed here: https://github.com/waku-org/js-waku/pull/2406